### PR TITLE
Make the metro bundle task to use for react-native-uifabric

### PR DESCRIPTION
**Pull request by dotnetjunky**
This PR implements a `Just` task, called metroPack, in order to produce the bundle for the `react-native-uifabric` package, and probably any packages that depends on react-native in the future.

### CODE REVIEW

File: experiments/react-native-uifabric/metro.config.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const uifabricsharedPackages = Object.keys(allPackages).reduce(
+  (prev, key) => (allPackages[key].packagePath.startsWith('packages') ? [...prev, key] : prev),
+  []
+);
+
+const additionalWatchFolders = uifabricsharedPackages.map(package => pathToSymLinkedPackage(package));
+
+const extraNodeModules = {
+  'react-native': path.resolve(__dirname, '../../node_modules/@office-iss/react-native-win32')
+};
```
Comment (acoates-ms) [inline experiments/react-native-uifabric/metro.config.js]: Can we just inline this?  seems odd to have it separate.

File: experiments/react-native-uifabric/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "typings": "lib/index.d.ts",
   "private": true,
   "scripts": {
-    "build": "just-scripts build --useMetro",
+    "build": "just-scripts build --useMetro --platform win32",
```
Comment (acoates-ms) [inline experiments/react-native-uifabric/package.json]: Do we really want to build all the bundles during build?

The fluent repo split things out into:
build
lint
test
bundle
as top level commands.  That way you dont have to sit there waiting for everything to bundle whenever you do a build.

File: experiments/react-native-uifabric/metro.config.js (hunk 1 above)
Comment (dotnetjunky) [reply experiments/react-native-uifabric/metro.config.js]: ok.

File: experiments/react-native-uifabric/package.json (hunk 2 above)
Comment (dotnetjunky) [reply experiments/react-native-uifabric/package.json]: OK, just pushed a new commit, which split the bundle task out of the build task.

**Comment by acoates-ms** [comment]
Should probably add to the CI loop a task that builds the bundles, to verify that they do not get broken.

**Pull request merged**